frontend/src/metabase/internal/components/ComponentsApp.jsx

```diff
@@ -101,9 +103,18 @@ export default class ComponentsApp extends Component {
                             <div className="p2 bordered flex align-center flex-full">
                               <div className="full">{element}</div>
                             </div>
-                            <div className="border-left border-right border-bottom text-code">
-                              <div className="p1">
-                                {reactElementToJSXString(element)}
+                            <div className="relative">
+                              <AceEditor
+                                value={reactElementToJSXString(element)}
+                                mode={"jsx"}
```

Comment:
I think this needs to be `ace/mode/jsx` according to https://github.com/metabase/metabase/blob/master/frontend/src/metabase/public/lib/code.js#L33